sdk/monitor/Azure.Monitor.Query/src/LogsQueryClient.cs

```diff
@@ -187,6 +191,110 @@ public virtual async Task<Response<LogsBatchQueryResult>> QueryBatchAsync(LogsBa
                 throw;
             }
         }
+
+        /// <summary>
+        /// Create a Kusto query from an interpolated string.  The interpolated values will be quoted and escaped as necessary.
+        /// </summary>
+        /// <param name="filter">An interpolated query string.</param>
+        /// <returns>A valid Kusto query.</returns>
+        public static string CreateQuery(FormattableString filter)
```

Comment:
Are verbatim strings supported also? ex. `$@"""quoted string {value}"""`

Reply:
Yes but not your example. In your example, the value is inserted inside the Kusto string literal. The expectation is that every formatting parameter is a distinct value of its own.